storage_service/locations/models/wellcome.py

```diff
@@ -133,22 +167,30 @@ def move_from_storage_service(self, src_path, dest_path, package=None):
             location = package.current_location
             space_id = location.relative_path.strip(os.path.sep)
 
+            # For reingests, the package status will still be 'uploaded'
+            # We want to use this to detect when upload is complete,
+            # so it is explicitly reset here.
+            package.status = Package.STAGING
+            package.save()
+
+            is_reingest = 'bag_id' in package.misc_attributes
             LOGGER.info('Callback will be to %s', callback_url)
             location = wellcome.create_s3_ingest(
                 space_id=space_id,
                 s3_key=s3_temporary_path,
                 s3_bucket=self.bucket_name,
                 callback_url=callback_url,
                 external_identifier=package.uuid,
-                ingest_type="create",
+                ingest_type="update" if is_reingest else "create",
             )
             LOGGER.info('Ingest_location: %s', location)
 
+            LOGGER.debug('Package status %s', package.status)
             while package.status == Package.STAGING:
                 # Wait for callback to have been called
                 for i in range(6):
                     package.refresh_from_db()
-                    print('Package status %s' % package.status)
+                    LOGGER.debug('Package status %s', package.status)
```

Comment:
👍 for logging

Tiny nit while we're improving this code: can we change the loop variable `i` to an underscore? It doesn't look like it's being used.

Reply:
It's not used :-) Unfortunately the underscore clashes with the `gettext` import (which I'm inclined to keep for consistency with other location modules). Not sure if there's some other alternative that would be better than just leaving `i` in place?